Yasgui/packages/yasgui/src/Tab.ts

```diff
@@ -105,6 +106,7 @@ export class Tab extends EventEmitter {
     this.rootEl.appendChild(wrapper);
     this.initControlbar();
     this.initYasqe();
+    this.emit("yasqeInitialized", this, this.getYasqe());
```

Comment:
The name "yasqeInitialized" is slightly different from its usage. It is fired on yasque init, when yasgue is drawn, when tab is changed. May be it can be more generic if we don't want to create many events. Something like "yasgueRenedered" or "yasqueReady".

Reply:
renamed to yasqeReady